colorpicker/src/main/java/org/catrobat/paintroid/colorpicker/ZoomableImageView.kt

```diff
@@ -312,9 +312,11 @@ class ZoomableImageView :
         updateLayoutParams {
             val point = Point()
             post {
-                display.getSize(point)
-                height = point.y
-                width = point.x
+                display?.let {
+                    it.getSize(point)
+                    height = point.y
+                    width = point.x
+                }
```

Comment:
```suggestion
                display?.getSize(point)
                height = point.y
                width = point.x
```
I think you can do the same thing by just adding a `?` 🤔
Also please change the year in the copyright section to 2022 :)
Just noticed that display.getSize() is deprecated in API 30, what do you think about replacing it with something similar to that mentioned in https://stackoverflow.com/questions/63719160/getsize-deprecated-in-api-level-30?

Reply:
True, I thought setting height and width should only be done if display is not null.
I added the alternative as described in the thread.

Reply:
Thanks for making the changes! Could you please fix the static analysis warnings, then I'll approve the PR :)

Reply:
Should be fixed now. I also checked the failing tests, none of them seems to be related to these changes.

Reply:
Could you please just check https://jenkins.catrob.at/job/Paintroid/job/PR-1040/6/testReport/junit/org.catrobat.paintroid.test.espresso.dialog/ColorDialogIntegrationTest/Tests___Device_Tests___alphaValueIsSetInSliderWhenChangedInSeekBar/ once locally? rest is fine :)

Reply:
It fails locally, but tried it with current develop branch without my changes and it fails too...

Reply:
hmm.. then I think this needs to be fixed maybe in a separate PR 🤔, I'll approve this one then.